basics/getting-started/frequent-questions/ingestion-faq.md

```diff
@@ -26,7 +26,30 @@ Pinot automatically detects new partitions in Kafka topics. It checks for new pa
 
 You can configure the interval for this job using the`controller.realtime.segment.validation.frequencyPeriod` property in the controller configuration.
 
-### How do I enable partitioning in Pinot, when using Kafka stream?
+### Does Pinot support multi-column partitioning?
+
+Pinot supports multi-column partitioning for offline tables. Map multiple columns under [`tableIndexConfig.segmentPartitionConfig.columnPartitionMap`](/configuration-reference/table.md#segmentPartitionConfig). When multi-column partitioning is configured, the configuration partitions the segment on each column individually. 
```

Comment:
Pinot does not partition the data. The input data needs to be partitioned already, and pinot can figure out which partition the input data belongs to based on the partition config.
We should also mention that each partition column is handled separately (e.g. the segment is partitioned on `memberId` and has partition id 1, also partitioned on `caseNumber` and has partition id 2)

Reply:
thanks! updated